go/cmd/dolt/commands/sqlserver/server.go

```diff
@@ -202,12 +203,13 @@ func Serve(
 		)
 	}
 
-	if startError != nil {
+	if errors.Is(startError, server.UnixSocketInUseError) {
+		lgr.Warn("unix socket set up failed: address already in use: listen unix at ", serverConf.Socket)
```

Comment:
(minor) just shortening up the error message a little bit... 

```suggestion
		lgr.Warn("unix socket set up failed: file already in use: ", serverConf.Socket)
```